### DISCUSSION TITLE
Still getting Codespace Prebuild Usage even though prebuilds have been disabled

### DISCUSSION THREAD

**Discussion by spenserblack** [community/community#48213] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

I generated a usage report, and, according to the report, [this repository](https://github.com/spenserblack/codespaces-emojicode) is still taking up usage for Prebuild Storage. However, I've double-checked the [settings in the repository](https://github.com/spenserblack/codespaces-emojicode/settings/codespaces), and confirmed that
>There are no prebuilds configured for this repository

Does anyone know why this repository is still showing up in my usage report for Prebuild Storage?

**Comment by jaysee260** [comment, marked as answer]
Hi @spenserblack 👋 . When did you disable prebuilds for this repo?

**Reply by spenserblack** [reply]
It must have been a month ago, or maybe more. When I discovered that prebuilds counted towards codespace storage usage, I went through several repos and disabled them.

The usage report that I generated was for the last seven days.

**Reply by jaysee260** [reply]
I looked into this, and it looks like for some reason, the prebuild template failed to be actually deleted when you disabled prebuilds for the repo, so usage was still being reported. I went ahead and deleted it on our end, so there should be no more usage reported from now. Thank you for bringing this up, and sorry for the inconvenience!

**Reply by spenserblack** [reply]
>sorry for the inconvenience!

No problem, thank you for helping me out!

**Reply by Erkansukgen** [reply]
dao

**Comment by yvan-sraka** [comment]
Hi GitHub Support team,

I believe I am experiencing a similar issue to the one discussed in this thread. Despite having removed all Codespace prebuilds linked to my account, I am still being billed for them. I’ve double-checked, and I am quite certain there are no prebuilds currently configured.

Additionally, I noticed an odd display: my top navigation bar shows 1 package, but when I click on it, there are no packages visible. It’s a bit confusing and possibly related.

It would be incredibly helpful to have a single page summarizing all active Codespaces prebuilds, rather than having to manually check each repository.

Could you please investigate why I’m still being billed for prebuild storage? Any help would be greatly appreciated!

Thanks for your time!

Best, Yvan

<img width="585" alt="Screenshot 2024-09-05 at 17 46 44" src="https://github.com/user-attachments/assets/a9d9b277-dc40-4d73-bcbc-64f50c6fc9d5">
